.storybook/webpack.config.js

```diff
@@ -1,8 +1,14 @@
 let babel = require('./babel.config.js');
 let config = require('./../.config/webpack/base.js')({
-    babel
+    babel,
+    preprocessor: {
+        variables: {
+            mode: 'eager'
```

Comment:
Storybook can't handle chunking too well - `eager` keeps the chunks in the entry-point instead